src/kili/services/label_response/response.py

```diff
@@ -0,0 +1,114 @@
+"""
+Classes for jobs response parsing.
+"""
+
+from typing import Dict, List
+
+from .annotation import Annotation, EntityAnnotation
+from .category import Category
+from .exceptions import (
+    AttributeNotCompatibleWithJobError,
+    JobNotExistingError,
+    MultipleCategoriesError,
+    NoCategoryError,
+)
+
+
+class JobResponse:
+    """
+    Class for job response parsing.
+    """
+
+    def __init__(self, job_name: str, job_response: Dict) -> None:
+        """
+        Class for job response parsing.
+
+        A job response can have the following keys:
+        - annotations
```

Comment:
missing arg docstring

Reply:
it's just below no?

Reply:
I mean: the text describing when `annotations` are filled